emulator.js

```diff
@@ -55,10 +54,10 @@ function (Cpu6502, Video, SoundChip, models, DdNoise, Cmos,  utils,fdc,tokeniser
         var end = 2358556;
         switch (mode) {
           case 3: case 6:
-            // This includes the "stripe" below the bottom line of the screen.
-            end = 2309468; break;
+          // This includes the "stripe" below the bottom line of the screen.
+          end = 2309468; break;
           case 7:
-            off = 263008; end = 2309468; break;
+          off = 263008; end = 2309468; break;
         }
```

Comment:
This seems to be unhelpfully flattening out indentation of code unrelated to the change.